qt/scientific_interfaces/test/ISISReflectometry/MainWindow/MainWindowPresenterTest.h

```diff
@@ -169,11 +178,12 @@ class MainWindowPresenterTest : public CxxTest::TestSuite {
   }
 
   void testShowOptionsOpensDialog() {
-    auto presenter = makePresenter();
-    // TODO Add test when options dialog is impelemented
-    // EXPECT_CALL(*m_optionsDialog, show()).Times(1);
+    auto optionsDialogPresenter =
+        std::make_unique<NiceMock<MockOptionsDialogPresenter>>();
+    EXPECT_CALL(*optionsDialogPresenter.get(), showView()).Times(AtLeast(1));
+    auto presenter = makePresenter(std::move(optionsDialogPresenter));
     presenter.notifyShowOptionsRequested();
-    verifyAndClear();
+    verifyAndClear(std::move(optionsDialogPresenter));
```

Comment:
`optionsDialogPresenter` has already been moved in the call to `makePresenter` so shouldn't be used again. Could you pass the raw pointer here? (Which you'd need to store in a local variable before the first move.)